libafl_qemu/src/snapshot.rs

```diff
@@ -521,7 +521,7 @@ where
     }
 }
 
-pub fn trace_write1_snapshot<QT, S>(
+pub fn trace_write_snapshot<QT, S, const N: usize>(
```

Comment:
I would say we should give this one a proper name (`PARAM_COUNT`?)

Reply:
The N was the width of the access so I matched it with the `trace_write_n_snapshot` parameter naming.